gradle-plugin/build.gradle

```diff
@@ -5,16 +5,13 @@ plugins {
 }
 apply plugin: 'kotlin'
 
-project.group = 'io.arrow-kt'
-project.version = '0.0.1'
```

Comment:
Sorry @jereksel , I'm looking forward to merging this pull request because it has a lot of good things, thanks!, but I've seen these lines right now. I think it's necessary [a way to manage the group and version for Gradle plugin](https://plugins.gradle.org/docs/publish-plugin). As far as I know it cannot be the same process we are following so far. A Gradle plugin can only be published once with a specific version. However, snapshots can be published without limit. Thanks!